src/components/FilterList.jsx

```diff
@@ -77,4 +89,10 @@ FilterList.propTypes = {
   apiEndpoint: PropTypes.string.isRequired,
 };
 
-export default FilterList;
+const FilterListWrapper = (props) => (
+  <Router>
```

Comment:
I think this is ok right now, but it could potentially cause some issues if this was embedded in it's own app. Something we should probably consider. Maybe making the routing optional, or more flexible would be a good direction to go. 

Will need to do some research on this.

Reply:
actually a practical example of this is pets. it is uses a router to navigate to details

Reply:
just a reminder, maybe we can avoid router at all here, and just use window. will test this.

Reply:
issue #20 tracks this issue, and will be addressed after this pr is completed.